### PR TITLE
gtk theming

### PR DESCRIPTION
**Pull request by Petingoso**
This is something I use and thought it was neat enough to share, the implementation is chaotic but it could very well be a nice improvement to the theme_change function
Also had to fixed gtk2 theming as its different
Thanks

### CODE REVIEW

File: .bin/theme_changer [hunk 1]
```diff
@@ -67,23 +67,32 @@ theme_change()
 	#bash $HOME/.bin/sudo_passhelper make -s -C ${dmenu_dir} install
 	#make -s -C ${dmenu_dir} clean
 }
+chgtk() {
```
Comment (junnunkarim) [inline .bin/theme_changer]: please change the name of the function to ```change_gtk_theme``` and move the curly brace to the next line.

File: .bin/theme_changer [hunk 2]
```diff
@@ -67,23 +67,32 @@ theme_change()
...
+   sed -e "s/^gtk-theme-name=.*/gtk-theme-name="$@"/" -i ~/.config/gtk-3.0/settings.ini
+   sed -e "s/^gtk-theme-name=.*/gtk-theme-name=\""$@"\"/" -i ~/.gtkrc-2.0 
+
```
Comment (junnunkarim) [inline .bin/theme_changer]: Please remove the empty line in ```line 73```

**Comment by junnunkarim** [comment]
Hello @Petingoso,
Thanks for this PR. I didn't knew you could change the gtk theme this way.
Please address the issues I mentioned and I will merge this PR.

**Comment by Petingoso** [comment]
Alright thanks for the feedback think it should be it

**Pull request merged**